content/en/logs/log_collection/csharp.md

```diff
@@ -311,6 +311,23 @@ New logs are now directly sent to Datadog.
 [1]: https://www.nuget.org/packages/Serilog.Sinks.Datadog.Logs
 [2]: https://app.datadoghq.com/account/settings#api
 [3]: /logs/#reserved-attributes
+{{% /tab %}}
+{{% tab "Log4Net" %}}
+
+You can manually configure a [RemotingAppender][1] to send logs to the Datadog API by adding to the web.config this:
+
+```
+<appender name="RemotingAppender" type="log4net.Appender.RemotingAppender" >
+    <sink value="https://http-intake.logs.datadoghq.com:433/v1/input/<DATADOG_API_KEY>" />
```

Comment:
We should recommend to add query parameter to set the source and service.

`https://http-intake.logs.datadoghq.com:433/v1/input/<DATADOG_API_KEY>?ddsource=csharp&service=<APPLICATION_NAME>`

Reply:
updated